### PR TITLE
Move the stopwatches to the eventsource stream

### PR DESCRIPTION
**Pull request by zeripath**
Use the /user/events eventsource to update the stopwatches
instead of polling /api/v1/user/stopwatches if the eventsource
is enabled.

Signed-off-by: Andrew Thornton <art27@cantab.net>

### CODE REVIEW

File: routers/events/events.go [hunk 1]
```diff
@@ -75,6 +81,32 @@ loop:
 		case <-shutdownCtx.Done():
 			go unregister()
 			break loop
+		case <-stopwatchTimer.C:
```
Comment (zeripath) [inline routers/events/events.go:84]: This could be a little more efficient and happen in the modules/eventsource instead per uid.

**Comment by zeripath** [comment]
I don't completely understand why this is polling in any case instead of looking for some change but moving the polling to /user/events will clean up the http log.

**Comment by 6543** [comment]
tested, if stopwatch is running with this patch the icon will disappear after a few seconds

**Comment by zeripath** [comment]
actually I think it wasn't a local issue on your side - it was a real issue. I've fixed it now. Feel free to test again @6543

**Review by 6543 — APPROVED** [review]
can confirm that it now works smoothly

**Review by lunny — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

**Pull request merged**